libs/forms/src/validators/not-empty.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  AbstractControl,
+  FormGroup,
+  ValidationErrors,
+  ValidatorFn,
+} from '@angular/forms';
+
+/**
+ * A validator for a form group that returns an error if all of its immediate children don't have a truthy value.
+ * Empty strings are considered falsy.
+ */
+// TODO: handle nested form groups
```

Comment:
Is this Todo still correct?

Reply:
yeah I think ideally this validator would handle nested form groups. Do you think it should be implemented for this PR? I don't have an immediate use case for it.